cookbook/migration-vuex.md

```diff
@@ -8,21 +8,21 @@ Primero, siga la [Guía de introducción](../getting-started.md) para instalar P
 
 ## Reestructuración de Módulos a Almacén
 
-Vuex tiene el concepto de un único almacén con múltiples _módulos_. Opcionalmente, estos módulos pueden tener espacios de nombres e incluso anidarse entre sí.
+Vuex tiene el concepto de un único almacén con múltiples _módulos_. Opcionalmente, estos módulos pueden tener namespaced e incluso anidarse entre sí.
 
-La forma más fácil de hacer la transición de ese concepto para usar con Pinia es que cada módulo que usó anteriormente ahora es un _almacén_. Cada almacén requiere de un `id` que es similar a un espacio de nombres en Vuex. Esto significa que cada almacén tiene un espacio de nombres por diseño. Los módulos anidados también pueden convertirse cada uno en su propia almacén. Los almacenes que dependen unas de otras simplemente importarán el otro almacén.
+La forma más fácil de hacer la transición de ese concepto para usar con Pinia es que cada módulo que usó anteriormente ahora es un _almacén_. Cada almacén requiere de un `id` que es similar a un namespace en Vuex. Esto significa que cada almacén tiene un namespaced por diseño. Los módulos anidados también pueden convertirse cada uno en su propia almacén. Los almacenes que dependen unas de otras simplemente importarán el otro almacén.
```

Comment:
Sure, I checked if there were other files with the same problem, but fortunately only this file had that type, because it was one of the first files that I tried to handle the translation of "store" as "almacen" and not as "tienda" Thank you for notifying me, I will always be careful